react/features/participants-pane/functions.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Check if two arrays are equal.
+ *
+ * @param {Array} arr1 - First array.
+ * @param {Array} arr2 - Second array.
+ * @returns {boolean} Are they equal or not.
+ */
+export const arrayEquality = (arr1, arr2) => {
+    if (arr1.length !== arr2.length) {
+        return false;
+    }
+
+    for (let i = 0; i < arr1.length; i++) {
+        if (arr1[i] !== arr2[i]) {
+            return false;
+        }
+    }
+
+    return true;
+};
+
+/**
+ * Generates a class attribute value.
+ *
+ * @param {Iterable<string>} args - String iterable.
+ * @returns {string} Class attribute value.
+ */
+export const classList = (...args) => args.filter(Boolean).join(' ');
+
+
+/**
+ * Find the first styled ancestor component of an element.
+ *
+ * @param {Element} target - Element to look up.
+ * @param {StyledComponentClass} component - Styled component reference.
+ * @returns {Element|null} Ancestor.
+ */
+export const findStyledAncestor = (target, component) => {
+    if (!target || target.matches(`.${component.styledComponentId}`)) {
+        return target;
+    }
+
+    return findStyledAncestor(target.parentElement, component);
+};
+
+/**
+ * Get a style property from a style declaration as a float.
+ *
+ * @param {CSSStyleDeclaration} styles - Style declaration.
+ * @param {string} name - Property name.
+ * @returns {number} Float value.
+ */
+export const getFloatStyleProperty = (styles, name) =>
+    parseFloat(styles.getPropertyValue(name));
+
+/**
+ * Gets the outer height of an element, including margins.
+ *
+ * @param {Element} element - Target element.
+ * @returns {number} Computed height.
+ */
+export const getComputedOuterHeight = element => {
+    const computedStyle = getComputedStyle(element);
+
+    return element.offsetHeight
+    + getFloatStyleProperty(computedStyle, 'margin-top')
+    + getFloatStyleProperty(computedStyle, 'margin-bottom');
+};
+
+/**
+ * Returns a renderable value from a boolean-like value - to be used in render
+ * methods where certain falsy values might get rendered inadvertently.
+ *
+ * @param {any} value - Value.
+ * @returns {any} Initial value or null.
+ */
+export const renderable = value => value || null;
```

Comment:
This is used in a single place, maybe you just need to apply Boolean() instead?

Reply:
Seems like I was trying to solve a problem that doesn't exist anymore... I had the impression that rendering anything other than `null` would cause the thing to get coerced into a string, resulting in "false" and "undefined" getting output into the DOM but it seems this not the case (anymore :s). Removed.